swagger-spec/licenses/detail.yaml

```diff
@@ -1 +1,29 @@
-# /licenses/{license_id}/
+parameters:
+  - in: path
+    type: string
+    required: true
+    name: license_id
+    description: 'The unique identifier of the license.'
+
+get:
+  summary: Retrieve a license
+  description: >-
+    Retrieves the details of a license.
+
+    #### Returns
+
+    Returns a dictionary with a `data` property containing the representation of the requested
+    license, if the request is successful.
+
+
+    If the request is unsuccessful, a dictionary with an `errors` property containing
+    information about the failure will be returned. Refer to the [list of error codes]()
+    to understand why this request may have failed.
+  tags:
+    - Licenses
+  operationId: licenses_read
+  responses:
```

Comment:
Between the `operationId` and `responses` keys, can you add a `x-response-schema` key with `License` as its value? This value is used to display the correct object name in the UI (https://github.com/caseyrollins/swagger-ui/commit/e482b22daa5c711f4f63a2c42f78106298d69365).

Also, fun fact, we can add whatever properties we want to the specification if they're prefixed with `x-`.